### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bwbohl**
copy and adapt from https://github.com/music-encoding/guidelines/blob/26990abf3523842592662cc949c2406f43d2dcdf/README.md

### CODE REVIEW

**Comment by bwbohl** [comment]
Since the source code of the guidelines were merged back into this repository and the https://github.com/music-encoding/guidelines repo has been reconfigured to only hold automatically generated HTML pages of the guidelines the contribution guidlines there is no longer needed. As this repo was is pointing to a contribution guideline in the guidelines repo it was best moved here, because contribution now happens here, for both schemata and guidelines.

**Comment by bwbohl** [comment]
@ahankinson @kepper @musicEnfanthen any concerns merging?

**Review by ahankinson — APPROVED** [review]
I'm assuming that all the directions are still correct, so no issues from me.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Contributing Guidelines
+
+If you wish to contribute changes to the MEI website, you can edit the files directly on GitHub. The text of the Guidelines is written in a dialect of [Markdown](https://kramdown.gettalong.org/quickref.html).
```
Comment (musicEnfanthen) [inline CONTRIBUTING.md:3]: Last sentence needs to be adapted, or isn't it needed at all?

**Review by musicEnfanthen — CHANGES_REQUESTED** [review]

**Comment by musicEnfanthen** [comment]
Thanks for the reminder! Small change request above.

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
Looks good. Thanks for getting this helpful information back in.

**Pull request merged**